src/App.css

```diff
@@ -1,4 +1,41 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
 .App {
-  display: grid;
+  text-align: center;
```

Comment:
Hi Evi,
`text-align` only centers text. if you want the child-elements centered, flex gives us the possibility to center them with `align-items`. :)

Reply:
nice, still not so used to flex 🥲